doc/recipe_elements/software_selection.rst

```diff
@@ -0,0 +1,24 @@
+========================
+Software Selection spoke
+========================
+
+/installation/hub/software_selection/addon
+==========================================
+Handles software addons (*Additional software for Selected Environment*) selection.
+
+Attributes:
+
+* ``action`` - ``select`` or ``deselect``
+* ``id`` - addon id (as defined in comps file) or glob pattern
+* ``subselect`` (optional)
+    * ``random`` - selects random subset of addons (``policy=just_check*`` conflicts
```

Comment:
Hmm, the asterisk is there on purpose, it should mean a 'glob' pattern (just_check, just_check_fail ...). I will try to tune it a bit, so that it's more obvious what I meant.

Reply:
I see, okay, it makes sense that way. Furthermore, I agree, that it will be better if you explain why it is there, as I was totally unaware of the reason.